src/prefect/client/orchestration/_deployments/client.py

```diff
@@ -50,7 +50,7 @@ def create_deployment(
         flow_id: UUID,
         name: str,
         version: str | None = None,
-        version_info: "VersionInfo | None" = None,
+        version_info: "Optional[VersionInfo]" = None,
```

Comment:
Is it necessary to use `Optional` here? I'd prefer to leave the `|` syntax so we don't need to import `Optional`.

Reply:
This is odd because I was getting errors in CI using `|` syntax in some places, so I reverted all of these back to `Optional`.  Let me see what happens if I go back to `|`

Reply:
Let's see if I still have trouble: https://github.com/PrefectHQ/prefect/pull/17466/commits/db6e307c8e3b13b177c86b863fe2d665147868c5

Reply:
https://github.com/PrefectHQ/prefect/actions/runs/14578804550/job/40890750170?pr=17466

Reply:
I _think_ I've got the right combo of 3.9 annotations and `from __future__ import annotations` now, let me know what you think!